build/build_docker.sh

```diff
@@ -0,0 +1,20 @@
+set -o nounset
+set -o pipefail
```

Comment:
Recommendation: `set -x` for the script to echo each command during execution to facilitate debugging. I don't (yet?) see sensitive values in the commands or output.

```suggestion
set -o pipefail
set -o xtrace
```